File: Python/Medium/oddish_or_evenish.py
# https://edabit.com/challenge/H3t4MkT9wGdL9P6Y3
def oddish_or_evenish(num):
    total=sum(int(i) for i in list(str(num)))
    return "Oddish" if total%2 else "Evenish"

print(oddish_or_evenish(43), "Oddish")
print(oddish_or_evenish(373), "Oddish")
print(oddish_or_evenish(55551), "Oddish")
print(oddish_or_evenish(694), "Oddish")
print(oddish_or_evenish(4433), "Evenish")
print(oddish_or_evenish(11), "Evenish")
print(oddish_or_evenish(211112), "Evenish")

File: javascript/Hard/oddishOrEvenish.js
// https://edabit.com/challenge/r6TSNwkLZ2DgsoKiH

function oddishOrEvenish(num) {
	const x = Array.from(String(num),Number).reduce((a,b)=> a+b,0)
    return x%2===0?'Evenish':'Oddish'
}


console.log((oddishOrEvenish(373), "Oddish"));
console.log((oddishOrEvenish(4433), "Evenish"));
